### ISSUE TITLE
Drop support for node < 4

### ISSUE THREAD

**Issue by jimmywarting** [zalmoxisus/remotedev-server#31]
object-assign is no longer needed for example

**Comment by zalmoxisus** [comment]
Yes, since [`µWS`](https://github.com/uWebSockets/uWebSockets) already requires at least node 4.x, we cannot support earlier version, and no need for that polyfill anymore.

I'll look into it, but in case you want to submit a PR with that change, it would be more than welcome.
